src/scripts/ordo_subsets.py

```diff
@@ -0,0 +1,84 @@
+"""Creates a ROBOT template to update mondo terms with the correct ORDO subset annotation."""
+from argparse import ArgumentParser
+
+import pandas as pd
+
+from utils import remove_angle_brackets
+
+
+SUBSET_MAP = {
+    'Orphanet:557493': 'http://purl.obolibrary.org/obo/mondo#ordo_disease',  # disorder
+    'Orphanet:557492': 'http://purl.obolibrary.org/obo/mondo#ordo_group_of_disorders',  # group of disorders
+    'Orphanet:557494': 'http://purl.obolibrary.org/obo/mondo#ordo_clinical_subtype',  # subtype of a disorder
```

Comment:
```suggestion
    'Orphanet:557494': 'http://purl.obolibrary.org/obo/mondo#ordo_subtype_of_a_disorder',  # subtype of a disorder
```

Wait for discussion in `#mondo-ingest` slack channel to conclude before accepting this.